easybuild/easyconfigs/l/libxml2/libxml2-2.9.3-foss-2016a-Python-2.7.11.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'ConfigureMake'
+
+name = 'libxml2'
+version = '2.9.3'
+
+homepage = 'http://xmlsoft.org/'
+description = """Libxml2 is the XML C parser and toolchain developed for the Gnome project (but usable
+ outside of the Gnome platform)."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+toolchainopts = {'pic': True}
+
+source_urls = [
+    'http://xmlsoft.org/sources/',
+    'http://xmlsoft.org/sources/old/'
+]
+sources = [SOURCELOWER_TAR_GZ]
+
+configopts = 'CC="$CC" CXX="$CXX" --with-pic --with-zlib=$EBROOTZLIB'
+
+pythonver = '2.7.11'
+pythonshortver = '.'.join(pythonver.split('.')[:2])
+versionsuffix = '-%s-%s' % ('Python', pythonver)
```

Comment:
@SimonPinches please use `%(pyver)s` here?

```python
name = 'libxml2'
version = '2.9.3'
versionsuffix = '-Python-%(pyver)s'
...
dependencies = [
    ('zlib', '1.2.8'),
    ('Python', '2.7.11'),
]
```

note that `pythonshortver` isn't used at all...